rules-tests/TypeDeclaration/Rector/ClassMethod/ReturnNeverTypeRector/Fixture/parent_protected.php.inc

```diff
@@ -5,8 +5,11 @@ namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\ReturnNeverTypeRector\
 use Rector\Core\Exception\ShouldNotHappenException;
 use Rector\Tests\TypeDeclaration\Rector\ClassMethod\ReturnNeverTypeRector\Source\vendor\SomeInterfaceWithReturnType;
 
-final class SkipParentProtected implements SomeInterfaceWithReturnType
+final class ParentProtected implements SomeInterfaceWithReturnType
```

Comment:
in most unit tests I just had to move from `:never` to the phpdoc equivalent.

this testcase is different. while php-src did not allow to add a `:never` in this case, this is supported when we use the phpdoc counterpart.

therefore this test was moved from a skip-test to a feature-test